docs-go/flows.md

```diff
@@ -148,7 +148,7 @@ first.
 - {Go}
 
   To deploy flows using Cloud Run and similar services, define your flows, and
-  then call `StartFlowServer()`:
+  then call `Init()`:
```

Comment:
why this change? i thought StartFlowServer was clearer but maybe it's not accurate. Init() sounds like you always need to call it -- do you?

Reply:
`Init` is not just the equivalent of `StartFlowServer`. It also starts the Reflection API server (previously a side effect of `StartFlowServer` which is not technically correct, it freezes the registry, and in the future will house all other Genkit initialization work that we might want to do.

So yes, you always need to call it. The equivalent in JS is `configureGenkit()` which must always be called and the flow server and Reflection API server will soon be started from there implicitly as well.